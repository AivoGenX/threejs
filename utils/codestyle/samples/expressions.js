
// GOOD

a + b;

a + b +
	c + d;

x ++;
++ x;
! x;

a ? b : c;

a + ( - b ) + c;

new ( a ? b : c )();
delete a[ 'prop' ];

obj[ 'key' ];
obj.key;

foo( aaa, bbb, ccc );
foo( a,   b,   c );
foo(   a,   b,   c );

// BAD

! x

x++;
++x;
!x;

a?b:c;
a ? b: c;

// unary sign operators should have space after them
-a + b + ( -c );

obj['key'];
obj[ 'key'];
obj['key' ];
