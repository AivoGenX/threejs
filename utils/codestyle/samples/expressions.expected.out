
// GOOD

a + b;

a + b +
	c + d;

x ++;
++ x;
! x

a ? b : c;

new ( a ? b : c )();
delete a;

obj[ 'key' ]
obj.key

// BAD

x ++;
++ x;
! x;

a ? b : c;
a ? b : c;

obj[ 'key' ]
obj[ 'key' ]
obj[ 'key' ]
